content/lxd/getting-started-cli.md

```diff
@@ -143,10 +143,14 @@ Instructions on building and installing LXD from source [can be found here](http
 
 Before you can create an instance, you need to configure LXD.
 
-Run the following as root:
+Run the following command as root to specify the configuration options:
 
     lxd init
 
+If you want to use the default options, add the `--auto` flag:
```

Comment:
Sry for the late comment, but I think it might not be such a good idea to encourage users to use the `auto` flag.

For those that want to use the default options, they can simply click enter during the configuration (if I remember correctly).

@stgraber

Reply:
Yeah, I think I agree with that, not necessarily because I want to force the users to go through `lxd init` but because `lxd init --auto` doesn't get to look at context.

So while `lxd init` will detect that your system is already on btrfs or ZFS and change its recommendation to use that, `lxd init --auto` just sends a fixed configuration to LXD, leading to less optimal configuration. The main one being that `lxd init --auto` will always result in the slower `dir` storage backend.

Reply:
@ru-fu what do you think?

Reply:
As a total newbie just wanting to get a working installation, I would have liked to know about the `--auto` option. Of course I can just accept all defaults - but this will make me think about each option, and the docs available in the table underneath are not sufficient to make smart decisions if you don't know what you're doing.
So for a quick getting started, I think the `--auto` is very helpful.

We could update to something like "To quickly create a test environment using the `dir` storage backend and the default configuration options, add the `--auto` flag:"?

Reply:
Okay, so it's probably fine to mention `lxd init --auto` but with the specific mention that this will result in a function but non-optimized setup, specifically sub-optimal storage (`dir` backend doesn't provide fast snapshots, fast copy/launch, no quotas, no optimized backups and slow migration).

Reply:
Ok, I will add the change to my PR #496. If no one objects.